onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -577,6 +579,10 @@ class PoolOpSupportChecker : public BaseOpSupportChecker {
       });
 }
 
+/* static */ bool PoolOpSupportChecker::IsQuantizedOp(const NodeUnit& node_unit) {
+  return IsQuantizedPool(GetQuantizedOpType(node_unit));
+}
```

Comment:
Can this call PoolOpBuilder::IsQuantizedOp or vice-versa instead of having 2 implementations of the same check?

Reply:
Keep it in PoolOpSupportChecker instead of PoolOpBuilder since OpBuilder will not be available in the non-android NNAPI build

Reply:
yes, I already kept it in Poolopsupportchecker. but not sure why the build breaks..because of this

Reply:
The PoolOpSupportChecker is not exposed in the header files, it will have to be kept in both places